### PR TITLE
fix manhwa.club's name

### PR DESCRIPTION
**Pull request by AriaMoradi**
Requested by Ichigo Kurosaki on Discord.

### CODE REVIEW

**Comment by arkon** [comment]
Backwards compatibility with the version pre-split? This will change the ID.

**Comment by AriaMoradi** [comment]
This time I tested it thoughtfully and made sure it works backwards compatible properly.

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt [hunk 1]
```diff
@@ -146,7 +146,7 @@ class MadaraGenerator : ThemeSourceGenerator {
             SingleLang("Manhwa Raw", "https://manhwaraw.com", "ko"),
             SingleLang("Manhwatop", "https://manhwatop.com", "en"),
             SingleLang("Manwahentai.me", "https://manhwahentai.me", "en", className = "ManwahentaiMe", isNsfw = true),
-            SingleLang("Manwha Club", "https://manhwa.club", "en"),
+            SingleLang("Manhwa.club", "https://manhwa.club", "en", className="ManwhaClub", overrideVersionCode = 1), // wrong class name for backward compatibility
```
Comment (arkon) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt:149]: I'm still confused as to why the class name needs to be wrong?

Comment (AriaMoradi) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt:149]: The gradle module is named after the pkgName which is named after className by default.
look at uses of `pkgNameSuffix`

**Pull request merged**